### PR TITLE
enhancement/user + logging fix

### PR DESCRIPTION
**Pull request by jwdotpark**
Changes
1. added get user API to query username parameter without token
2. fixed logger token, it prints log with timestamp on both *.log and console

It'd be nice if you can check side by side with updated [api doc ](https://app.swaggerhub.com/apis-docs/Goodbuy-node/Goodbuy/1.0.0#/)!

### CODE REVIEW

File: src/controllers/user.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import UserModel from "../models/user.model";
+import { Request, Response, NextFunction } from "express";
+
+export const getOtherProfile = (req: Request, res: Response, next: NextFunction) => {
+  UserModel.findOne({ username: req.params.username })
```
Comment (Darjusch) [inline src/controllers/user.ts:5]: Validate the input before you query please :D There is a lot of examples in the validator folder.

File: test/profile.js [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-  createRefreshToken,
   createAccessToken,
 } = require("../dist/controllers/auth");
 let chai = require("chai");
 let chaiHttp = require("chai-http");
 let chaiCookie = require("chai-expected-cookie");
 let server = require("../dist/server");
-let expect = chai.expect;
-let jwt = require("jsonwebtoken");
+// let expect = chai.expect;
```
Comment (Darjusch) [inline test/profile.js:9]: Not something big but try to avoid pushing "dead code"

**Review by Darjusch — CHANGES_REQUESTED** [review]
After the changes you can merge

File: src/controllers/user.ts (hunk 1 above)
Comment (jwdotpark) [reply src/controllers/user.ts]: image 
hope this works for me, i always forget to add :D validation logic is added, sorry about that

**Pull request merged**